names.go

```diff
@@ -70,6 +70,12 @@ func (r FutureNameShowResult) Receive() (*ncbtcjson.NameShowResult, error) {
 //
 // See NameShow for the blocking version and more details.
 func (c *Client) NameShowAsync(name string, options *ncbtcjson.NameShowOptions) FutureNameShowResult {
+	if options == nil {
+		options = &ncbtcjson.NameShowOptions{}
+	}
+	if options != nil {
```

Comment:
I think you don't need to check for nil here since you fix that above?

Reply:
If I'm not wrong, I believe it still triggered the mandatory Go lint errors, but retrying